docs(cli): clarify on what 'nest build' does (#2440)
nestjs/docs.nestjs.com: content/cli/usages.md

Compiles an application or workspace into an output folder.

> Besides that, the `build` command is responsible for:

Reviewer: ```suggestion
Also, the `build` command is responsible for:
```

> - mapping paths properly for us (if using path aliases) via `tsconfig-paths`

Reviewer: ```suggestion
- mapping paths (if using path aliases) via `tsconfig-paths`
```

> - map paths properly for us (if using path aliases) via `tsconfig-paths`;
> - annotate DTOs with swagger decorators (if `@nestjs/swagger` is enable);
> - annotates DTOs with GraphQL decorators (if `@nestjs/graphql` is enable).

Reviewer 2: ```suggestion
- mapping paths properly for us (if using path aliases) via `tsconfig-paths`
- annotating DTOs with OpenAPI decorators (if `@nestjs/swagger` is enable)
- annotating DTOs with GraphQL decorators (if `@nestjs/graphql` is enable)
```

Following better grammar, "X is responsible for _doing_ Y"

Reviewer 2: Also, `OpenAPI` vs Swagger for the "proper" name. Feel free to ignore that if we generally use swagger elsewhere in the docs

> - annotating DTOs with OpenAPI decorators (if `@nestjs/swagger` plugin is enable)
> - annotating DTOs with GraphQL decorators (if `@nestjs/graphql` plugin is enable)

Reviewer: ```suggestion
- annotating DTOs with OpenAPI decorators (if `@nestjs/swagger` plugin is enabled)
- annotating DTOs with GraphQL decorators (if `@nestjs/graphql` plugin is enabled)
```
